src/frontend/src/lib/services/auth.services.ts

```diff
@@ -115,6 +115,11 @@ const clearTestnetsOption = async () => {
 	testnetsStore.reset({ key: 'testnets' });
 };
 
+// eslint-disable-next-line require-await
+const clearSessionStorage = async () => {
+	sessionStorage.clear();
```

Comment:
Why are we clearing the session storage?

Reply:
@peterpeterparker Because `sveltes` session storage is cleared when the page session ends. This only happens if the browser or the tab is closed. This means that we have to handle the session storage clear on a logout.